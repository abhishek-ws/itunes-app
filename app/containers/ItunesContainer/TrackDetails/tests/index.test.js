/**
 *
 * Tests for TrackDetails
 *
 *
 */

import React from 'react';
import { renderProvider } from '@utils/testUtils';
// import { fireEvent } from '@testing-library/dom'
import { TrackDetailsTest as TrackDetails } from '../index';
import ReturnWithRouter from '@utils/returnRouter';
import getIntl from '@utils/createIntl';

describe('<TrackDetails /> container tests', () => {
  let submitSpy;

  beforeEach(() => {
    submitSpy = jest.fn();
  });
  it('should render and match the snapshot', () => {
    const { baseElement } = renderProvider(
      <ReturnWithRouter>
        <TrackDetails dispatchTrackSearch={submitSpy} dispatchClearTrackDetails={submitSpy} />
      </ReturnWithRouter>
    );
    expect(baseElement).toMatchSnapshot();
  });
});
