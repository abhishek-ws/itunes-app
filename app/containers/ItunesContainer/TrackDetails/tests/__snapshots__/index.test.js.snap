// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TrackDetails /> container tests should render and match the snapshot 1`] = `
<body>
  <div>
    <div
      class="TrackDetails__Container-dlhfxi-0 iuNvlR"
    >
      <div
        class="ant-skeleton ant-skeleton-active"
      >
        <div
          class="ant-skeleton-content"
        >
          <h3
            class="ant-skeleton-title"
            style="width: 38%;"
          />
          <ul
            class="ant-skeleton-paragraph"
          >
            <li />
            <li />
            <li
              style="width: 61%;"
            />
          </ul>
        </div>
      </div>
      <a
        href="/"
      >
        <p
          class="T__StyledText-gjlic1-0 kYcyRq TrackDetails__StyledT-dlhfxi-1 gRXuBf"
          data-testid="t"
        >
          Back to home
        </p>
      </a>
    </div>
  </div>
</body>
`;
